src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Sse41.cs

```diff
@@ -255,6 +255,11 @@ public new abstract class X64 : Sse2.X64
         /// </summary>
         public static byte Extract(Vector128<byte> value, byte index) => Extract(value, index);
         /// <summary>
+        /// int _mm_extract_epi8 (__m128i a, const int imm8)
+        ///   PEXTRB reg/m8, xmm, imm8
+        /// </summary>
+        public static byte Extract(Vector128<sbyte> value, byte index) => Extract(value, index);
```

Comment:
(super nit) - all the other overloads has the "signed" version listed first, then the "unsigned" version. Do we want to keep it consistent to have the signed version first here?